src/__tests__/__snapshots__/App.spec.jsx.snap

```diff
@@ -1310,6 +1310,106 @@ exports[`initial state renders without crashing 1`] = `
               All
             </button>
           </section>
+          <span
+            className="separator"
+          />
+          <section
+            className="chart-toggles m-btn-group"
+          >
+            <p>
+              Chart type
+            </p>
+            <button
+              className="a-btn toggle line selected"
```

Comment:
`toggle`, `line`, `selected` … these class names 😬

Reply:
to be cleaned up when you migrate everything to design system?